src/lib/skilling/skills/hunter/defaultBirdHouseTrap.ts

```diff
@@ -0,0 +1,16 @@
+export interface BirdhouseData {
+	lastPlaced: string | null;
```

Comment:
are you using the name of the birdhouse instead of ID?

Reply:
the name is the ID in this case